plugins/de.ovgu.featureide.fm.ui/src/de/ovgu/featureide/fm/ui/editors/featuremodel/editparts/FeatureEditPart.java

```diff
@@ -122,7 +123,7 @@ public void performRequest(Request request) {
 		if (request.getType() == RequestConstants.REQ_DIRECT_EDIT) {
 			showRenameManager();
 		} else if (request.getType() == RequestConstants.REQ_OPEN) {
-			final SetFeatureToCollapseOperation op = new SetFeatureToCollapseOperation(feature, featureModel);
+			final CollapseFeatureOperation op = new CollapseFeatureOperation(feature, featureModel, "Collapse Operation"); //TODO: PLATZHALTER
```

Comment:
"TODO: Platzhalter"?
Shouldn't the operationLabel ("Collapse Operation") be read from the Stringtable?

Reply:
fixed